src/types/onyx/Policy.ts

```diff
@@ -0,0 +1,32 @@
+import {ValueOf} from 'type-fest';
+import CONST from '../../CONST';
+import * as OnyxCommon from './OnyxCommon';
+
+type Policy = {
+    /** The ID of the policy */
+    id?: string;
+
+    /** The name of the policy */
+    name?: string;
+
+    /** The current user's role in the policy */
+    role?: ValueOf<typeof CONST.POLICY.ROLE>;
+
+    /** The policy type */
+    type?: ValueOf<typeof CONST.POLICY.TYPE>;
+
+    /** The email of the policy owner */
+    owner?: string;
+
+    /** The output currency for the policy */
+    outputCurrency?: string;
+
+    /** The URL for the policy avatar */
+    avatar?: string;
+
+    /** Error objects keyed by field name containing errors keyed by microtime */
+    errorFields?: Record<string, Record<string, string>>;
```

Comment:
Shouldn't here use `number`?

```suggestion
    errorFields?: Record<number, Record<string, string>>;
```


Because on `withPolicy.js`, the example on this field says this:
```
        /**
         * Error objects keyed by field name containing errors keyed by microtime
         * E.x
         * {
         *     name: {
         *        [DateUtils.getMicroseconds()]: 'Sorry, there was an unexpected problem updating your workspace name.',
         *     }
         * }
         */
          errorFields: PropTypes.objectOf(PropTypes.objectOf(PropTypes.string)),

```

And `DateUtils.getMicroseconds()` returns a number

Reply:
Bookmark 🔖 , continuing from here tomorrow

Reply:
I noticed that in some places it was string in others it was number, so I just went with string as it also accepts numbers. However number may be more accurate here. Wdyt @fabioh8010 @hayata-suenaga?

From guidelines:
> Even though `string` is specified as a key, `Record<string, unknown>` type can still accepts objects whose keys are numbers. This is because numbers are converted to strings when used as an object index. Note that you cannot use [symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) for `Record<string, unknown>`.

Reply:
Makes sense to me 👍🏻